test/integration/apis/constants.test.ts

```diff
@@ -0,0 +1,140 @@
+import { ApiPromise } from "@polkadot/api";
+import { assert } from "chai";
+import { ConstantsAPI, DefaultConstantsAPI } from "../../../src/apis/constants";
+import { createPolkadotAPI } from "../../../src/factory";
+import { defaultEndpoint } from "../../config";
+
+describe("Constants", function () {
+    this.timeout(10000); // API can be slightly slow
+
+    let api: ApiPromise;
+    let constantAPI: ConstantsAPI;
+
+    beforeEach(async () => {
+        api = await createPolkadotAPI(defaultEndpoint);
+        constantAPI = new DefaultConstantsAPI(api);
+    });
+
+    afterEach(async () => {
+        await api.disconnect();
+    });
+
+    describe("getDotExistentialDeposit", () => {
+        it("should sucessfully return", async () => {
+            const returnValue = await constantAPI.getDotExistentialDeposit();
+            assert.isDefined(returnValue);
```

Comment:
Maybe we can have slightly better tests, for example at least checking the type, and maybe things like greater than 0 or so?